### PR TITLE
Specify account and service names

### PR DESCRIPTION
**Pull request by joshcooper**
Describes what accounts and services are created by the `puppetserver`
and `puppet-agent` packages, and which account each service runs as.

### CODE REVIEW

**Comment by haus** [comment]
:+1:

File: file_paths.md [hunk 1]
```diff
@@ -121,6 +124,10 @@ when installing puppet-agent 32-bit on 64-bit windows, the
 installation path defaults to `C:\Program Files (x86)\Puppet
 Labs`. The examples below assume 2008/2012 and puppet-agent (64-bit).
 
+The package will create two services `puppet` and `mcollective`
+running as `LocalSystem` by default. It will not create a `puppet`
+user or group.
+
```
Comment (ferventcoder) [inline file_paths.md]: Do we need to specify what it used to be in older versions?

Comment (joshcooper) [inline file_paths.md]: Good question. For FOSS, they are the same (I think). For PE I think everything was prefixed with `pe-`. /cc @haus ?

**Comment by kylog** [comment]
Disregard my comment on "puppet-server" vs "puppetserver". I asked in the Puppet Server room and ... it was confusing. Not something we need to settle in this PR.

File: file_paths.md (hunk 1 above)
Comment (haus) [inline file_paths.md]: For PE, it used to be `pe-puppet` for both user/group and service name, and `pe-mcollective` for the mco service. For FOSS, the puppet user/group used to be created by the puppet package and the puppet user was used in the puppet-server and puppetmaster packages/services.

**Comment by joshcooper** [comment]
@jeffmccune @camlow325 do you have any comments before merging?

**Comment by camlow325** [comment]
Looks okay to me.

**Pull request merged**